### ISSUE TITLE
Tile encoding with Prov-GigaPath - Problem with input dimension

### ISSUE THREAD

**Issue by encinear** [prov-gigapath/prov-gigapath#103]
I am following the steps outlined in the HuggingFace link below to extract feature vectors from pathology images. However, I am receiving a warning that the size of input image must be 518 in order to obtain the feature vectors. Yet, in the tile encoding section, you are using 224-sized images.

https://huggingface.co/prov-gigapath/prov-gigapath

AssertionError: Input height (224) doesn't match model (518)

Do I need to resize my images as (518,518) ? Isn't it too big to process thousands of images ?

Could you please explain this situation?

**Comment by HanwenXuTHU** [comment]
Hi, did you use our default image transformation to process imaged before feeding them to the tile encoder?


`transform = transforms.Compose(
    [
        transforms.Resize(256, interpolation=transforms.InterpolationMode.BICUBIC),
        transforms.CenterCrop(224),
        transforms.ToTensor(),
        transforms.Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
    ]
)`

**Comment by encinear** [comment]
Yes, I exactly used the default image transformation for preprocessing before tile encoding;

import timm
from PIL import Image
from torchvision import transforms
import torch

tile_encoder = timm.create_model("hf_hub:prov-gigapath/prov-gigapath", pretrained=True)

transform = transforms.Compose(
    [
        transforms.Resize(256, interpolation=transforms.InterpolationMode.BICUBIC),
        transforms.CenterCrop(224),
        transforms.ToTensor(),
        transforms.Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
    ]
)


The model is asking images with (518,518)  size as input, (224,224) size does not work. Have you ever check recently tile encoding process ? 

Best.
